File: palindrome_checker.py
def is_palindrome(word):
    lower = word.lower()
    return lower == lower[::-1]

if __name__ == "__main__":
    test_words = ["madam", "Sir", "racecar", "Level", "hello"]
    for word in test_words:
        print(f"{word} - {is_palindrome(word)}")

File: palindromeChecker.js
function isPalindrome(word) {
    const lower = word.toLowerCase();
    return lower === lower.split('').reverse().join('');
}

const testWords = ["madam", "Sir", "racecar", "Level", "hello"];
for (const word of testWords) {
    console.log(`${word} - ${isPalindrome(word)}`);
}
